
# This is a patch for Devel-Cover-0.65 to update it to Devel-Cover-0.65-patched
# 
# To apply this patch:
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'applypatch' program with this patch file as input.
#
# If you do not have 'applypatch', it is part of the 'makepatch' package
# that you can fetch from the Comprehensive Perl Archive Network:
# http://www.perl.com/CPAN/authors/Johan_Vromans/makepatch-x.y.tar.gz
# In the above URL, 'x' should be 2 or higher.
#
# To apply this patch without the use of 'applypatch':
# STEP 1: Chdir to the source directory.
# If you have a decent Bourne-type shell:
# STEP 2: Run the shell with this file as input.
# If you don't have such a shell, you may need to manually delete
# the files as shown below.
# STEP 3: Run the 'patch' program with this file as input.
#
# These are the commands needed to create/delete files/directories:
#
rm -f ".gitignore"
#
# This command terminates the shell and need not be executed manually.
exit
#
#### End of Preamble ####

#### Patch data follows ####
diff -c "Devel-Cover-0.65\MANIFEST" "Devel-Cover-0.65-patched\MANIFEST"
Index: ./MANIFEST
*** ./MANIFEST	Fri Aug  7 17:49:00 2009
--- ./MANIFEST	Thu Feb 11 08:44:01 2010
***************
*** 1,4 ****
- .gitignore
  all_versions
  BUGS
  buildperl
--- 1,3 ----
#### End of Patch data ####

#### ApplyPatch data follows ####
# Data version        : 1.0
# Date generated      : Thu Feb 11 08:45:29 2010
# Generated by        : makepatch 2.04
# Recurse directories : Yes
# Excluded files      : (\A|/).*\~\Z
#                       (\A|/).*\.a\Z
#                       (\A|/).*\.bak\Z
#                       (\A|/).*\.BAK\Z
#                       (\A|/).*\.elc\Z
#                       (\A|/).*\.exe\Z
#                       (\A|/).*\.gz\Z
#                       (\A|/).*\.ln\Z
#                       (\A|/).*\.o\Z
#                       (\A|/).*\.obj\Z
#                       (\A|/).*\.olb\Z
#                       (\A|/).*\.old\Z
#                       (\A|/).*\.orig\Z
#                       (\A|/).*\.rej\Z
#                       (\A|/).*\.so\Z
#                       (\A|/).*\.Z\Z
#                       (\A|/)\.del\-.*\Z
#                       (\A|/)\.make\.state\Z
#                       (\A|/)\.nse_depinfo\Z
#                       (\A|/)core\Z
#                       (\A|/)tags\Z
#                       (\A|/)TAGS\Z
# r ".gitignore" 121 0
# p "MANIFEST" 5232 1265903041 0100666
#### End of ApplyPatch data ####

#### End of Patch kit [created: Thu Feb 11 08:45:29 2010] ####
#### Patch checksum: 46 1535 21452 ####
#### Checksum: 75 2590 44823 ####
